ext/adaptors/ranges-lib.tex

```diff
@@ -144,7 +145,8 @@
 \begin{codeblock}
 template <class T>
 concept bool ViewableRange =
-  Range<T> && (is_lvalue_reference_v<T> || View<decay_t<T>>); // \seebelow
+  Range<T> && (is_lvalue_reference_v<T> ||
+    @\added{(}@View<decay_t<T>> @\added{\&\& Constructible<decay_t<T>, T>)}@); // \seebelow
```

Comment:
This is covered by `View`, which requires `Semiregular`.

Reply:
`View<decay_t<volatile subrange<int*>>>` is satisfied, but `Constructible<subrange<int*>, volatile subrange<int*>>` is not.

Reply:
I really can't be bothered with `volatile`, and I don't want to complicate our interfaces to account for it.

Reply:
> I really can't be bothered with volatile, and I don't want to complicate our interfaces to account for it.

I understand that you don't like volatile, but we can't simply pretend that parts of the core language do not exist. Would you prefer blanket wording that forbids instantiating a library template with a type in which `volatile` participates in any way?

Reply:
> we can't simply pretend that parts of the core language do not exist

...despite that LWG has been doing exactly that with `volatile` for years, giving rise to the nasty C++17 NB comment suggesting we should either handle `volatile` properly or forbid it outright.

Reply:
I prefer to follow LEWG's example and ignore it. When they make a library-wide decision about how to handle it, we can follow suit.

Reply:
Reverted.